### PR TITLE
[mesheryctl] Display Loadbalancer IP for mesheryctl system status

### PR DESCRIPTION
**Pull request by Revolyssup**
…atus

Signed-off-by: ashish <ashishjaitiwari15112000@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Revolyssup** [comment]
@leecalcote  After this I can use mesheryctl instead of kubectl in one more place on the core workflow

**Comment by navendu-pottekkat** [comment]
@Revolyssup Why an additional function? Shouldn't this be the IP used in the meshconfig file as well?

File: mesheryctl/internal/cli/root/system/status.go [hunk 1]
```diff
@@ -178,7 +178,16 @@ var statusCmd = &cobra.Command{
 			// Print the data to a table for readability
 			utils.PrintToTable([]string{"Name", "Ready", "Status", "Restarts", "Age"}, data)
 
+			mesherySvc, err := utils.GetMesheryService(client, utils.MesheryNamespace)
+			if err != nil {
+				return err
+			}
 			log.Info("\nMeshery endpoint is " + mctlCfg.Contexts[mctlCfg.CurrentContext].Endpoint)
+			if len(mesherySvc.Status.LoadBalancer.Ingress) != 0 {
+				log.Info("\nLoadbalancer IP for Meshery: ", mesherySvc.Status.LoadBalancer.Ingress[0].IP)
```
Comment (bariqhibat) [inline mesheryctl/internal/cli/root/system/status.go:187]: Do we always have one ingess in the Meshery? Is there not a case where this is more than one item?

File: mesheryctl/internal/cli/root/system/status.go [hunk 2]
```diff
@@ -178,7 +178,16 @@ var statusCmd = &cobra.Command{
...
+				return nil
+			}
+			log.Warn("\nLoadbalancer IP not allocated for Meshery. You cannot connect to Meshery from outside the cluster. Make sure your ingress/loadbalancer is running.")
```
Comment (bariqhibat) [inline mesheryctl/internal/cli/root/system/status.go:190]: Not really sure about this, but can this be replaced with Meshkit errors?

**Review by bariqhibat — COMMENTED** [review]
Few comments!

**Comment by Revolyssup** [comment]
> @Revolyssup Why an additional function? Shouldn't this be the IP used in the meshconfig file as well?

Tbh I looked at the meshkit function being used. And that's not quite correct. It assumes a lot of things. Plus the way it is used as well is not very correct. I can talk about it in some meeting later. But I wanted a quick fix so did this. For now I'm closing this PR

**Pull request closed without merging**